Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -148,10 +148,15 @@ public class PythonCompiler : SinglePassCompiler {
     }
 
     protected override IClassWriter CreateClass(string moduleName, string name, bool isExtern, string fullPrintName,
-        List<TypeParameter> typeParameters, TopLevelDecl cls, List<Type> superClasses, IToken tok, ConcreteSyntaxTree wr) {
-      var methodWriter = wr.NewBlockPy(header: $"class {IdProtect(name)}:");
-
-      var needsConstructor = cls is TopLevelDeclWithMembers decl && decl.Members.Any(m => !m.IsGhost && m is Field && !m.IsStatic);
+      List<TypeParameter> typeParameters, TopLevelDecl cls, List<Type> superClasses, IToken tok, ConcreteSyntaxTree wr) {
+      var realSuperClasses = superClasses?.Where(trait => !trait.IsObject).ToList() ?? new List<Type>();
```

Comment:
`ToList()` will never return null, will it?

Reply:
Ah but don't you need this then?

```suggestion
      var realSuperClasses = superClasses?.Where(trait => !trait.IsObject)?.ToList() ?? new List<Type>();
```

Reply:
But wouldn't that fail on the first question mark?

Reply:
Ah never mind, I didn't realize `?.` was short-circuiting enough to avoid evaluating any more of the chain. :)